process.rb

```diff
@@ -136,6 +136,25 @@ def slugify(word)
     .where(election_name: election_name)
     .includes(:office_election, :calculations)
     .find_each do |candidate|
+      committee = Committee.where(Filer_ID: candidate.FPPC.to_s).first
+      contributions = candidate.calculation(:contributions_by_type)
+      total_contributions = candidate.calculation(:total_contributions)
+
+      # Calculate the proprtion of small contributions
+      unless committee.nil? || contributions.nil? || total_contributions == 0
+        total_small = committee.calculation(:total_small) + contributions['Unitemized']
+        candidate.save_calculation(:small_contributions, total_small)
```

Comment:
I'd suggest then renaming this one to become: `total_small_contributions` (since it includes unitemized + itemized contributions)

Reply:
done